recipes/python-symengine/bld.bat

```diff
@@ -0,0 +1 @@
+python setup.py install --symengine-dir=%LIBRARY_PREFIX%
```

Comment:
Please add these arguments to your install command `--single-version-externally-managed --record record.txt`.